ingest/Snakefile

```diff
@@ -23,41 +23,32 @@ TAXON_ID = config["taxon_id"]
 SEGMENTED = config["segmented"]
 LOG_LEVEL = config.get("log_level", "INFO")
 NCBI_API_KEY = os.getenv("NCBI_API_KEY")
-FILTER_FASTA_HEADERS = config.get("filter_fasta_headers", None)
 APPROVE_TIMEOUT_MIN = config.get("approve_timeout_min")  # time in minutes
 CHECK_ENA_DEPOSITION = config.get("check_ena_deposition", False)
 ALIGN = True
-GROUPS_OVERRIDE_JSON = None
+FILTER = config.get("metadata_filter", None)
+GROUPS_OVERRIDE_JSON = config.get("grouping_override", None)  # JSON map from group to segments' insdcAccessionFull
 
 dataset_server_map = {}
 dataset_name_map = {}
 
```

Comment:
If segmented, 2 options: use nextclade sort, or align against everything: Align can be tuned to recognize even short segments of diverse sequences, whereas sort is faster

ALIGN holds the method we use in this case, maybe clearer as enum of `segment_detection_method`